doc/book/box/indexes.rst

```diff
@@ -162,78 +167,86 @@ then it also changes the index keys defined for the tuple.
 
 **There are the following SELECT variations:**
 
-#.  The search can use comparisons other than equality:
+* The search can use **comparisons** other than equality:
 
-    ..  code-block:: tarantoolsession
+  ..  code-block:: tarantoolsession
 
-        tarantool> box.space.tester:select(1, {iterator = 'GT'})
-        ---
-        - - [2, 'Scorpions', 2015, 4]
-          - [3, 'Ace of Base', 1993]
-          - [4, 'Roxette', 2016, 3]
-        ...
+      tarantool> box.space.tester:select(1, {iterator = 'GT'})
+      ---
+      - - [2, 'Scorpions', 2015, 4]
+        - [3, 'Ace of Base', 1993]
+        - [4, 'Roxette', 2016, 3]
+      ...
 
-    The :ref:`comparison operators <box_index-iterator-types>` are LT, LE, EQ,
-    REQ, GE, GT (for "less than", "less than or equal", "equal", "reversed equal",
-    "greater than or equal", "greater than" respectively).
-    Comparisons make sense if and only if the index type is TREE.
+  The :ref:`comparison operators <box_index-iterator-types>` are:
 
-    Note that we didn't use the name of the index, which means we use primary index here.
+  *   ``LT`` for "less than"
+  *   ``LE`` for "less than or equal"
+  *   ``GT`` for "greater"
+  *   ``GE`` for "greater than or equal" .
+  *   ``EQ`` for "equal",
+  *   ``REQ`` for "reversed equal"
 
-    This type of search may return more than one tuple; if so, the tuples will be
-    in descending order by key when the comparison operator is LT or LE or REQ,
-    otherwise in ascending order.
+  Value comparisons make sense if and only if the index type is TREE.
+  The iterator types for other types of indexes are slightly different and work
+  differently. See details in section :ref:`Iterator types <box_index-iterator-types>`.
 
-#.  The search can use a **secondary index**.
+  Note that we didn't use the name of the index, which means we use primary index here.
 
-    For a primary-key search, it is optional to specify an index name as
-    was demonstrated above.
-    For a secondary-key search, it is mandatory.
+  This type of search may return more than one tuple; if so, the tuples will be
+  in descending order by key when the comparison operator is LT or LE or REQ,
+  otherwise in ascending order.
```

Comment:
Let's make two sentences here. https://www.tarantool.io/en/doc/latest/contributing/docs/style/#express-one-idea-in-a-sentence

```suggestion
  This type of search may return more than one tuple. The tuples will be sorted
  in descending order by key if the comparison operator is LT or LE or REQ.
  Otherwise they will be sorted in ascending order.
```

Also, will it be a tuple of tuples? A list?